content/docs/concepts/traffic-management/routing/service-entries/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Service Entries
+description: Learn about using a service entry to add an entry to Istio's abstract model that configures routing rules for external dependencies of the mesh.
+weight: 4
+keywords: [traffic-management, http, tcp, grpc, ingress, egress]
+---
+
+A [service entry](/docs/reference/config/networking/v1alpha3/service-entry)
+is a network resource used to add an entry to Istio's abstract model, or
+service registry, that Istio maintains internally. Once added, the Envoy
+proxies can send traffic to the external service as if it was a service in your
+mesh. You can configure service entries to configure routing rules to:
+
+-  Redirect and forward traffic for external destinations, such as APIs
+   consumed from the web, or traffic to services in legacy infrastructure.
+
+-  Define
+   [retry](/docs/concepts/traffic-management/network/#timeouts-and-retries),
+   [timeout](/docs/concepts/traffic-management/network/#timeouts-and-retries),
+   and [fault injection](/docs/concepts/traffic-management/network/#fault-injection)
+   policies for external destinations.
+
+- Add a service running in a Virtual Machine to the mesh to [expand your mesh](/docs/setup/kubernetes/additional-setup/mesh-expansion/#running-services-on-a-mesh-expansion-machine).
+
+- Logically add services from a different cluster to the mesh to configure a
+  [multicluster Istio mesh](/docs/tasks/multicluster/gateways/#configure-the-example-services)
+  on Kubernetes.
+
+You don’t need to add a service entry for every mesh-external service that you
+want your mesh services to use. By default, Istio configures the Envoy proxies
+to passthrough requests from unknown services. You can also use service entries
+to configure internal infrastructure:
+
+-  A **mesh-internal** service entry adds a service running in unmanaged
+   infrastructure, which does not have an Envoy proxy, but still is within the
+   same network, to the mesh. You configure mesh-internal service entries as
+   you expand the mesh to include unmanaged infrastructure.
+
+    For example, you create mesh-internal service entries for the services
+    running on VMs outside the Kubernetes cluster but within the network.
+    Mutual TLS authentication (mTLS) is enabled by default for mesh-internal
+    service entries, but to change the authentication method, you can configure
+    a destination rule for the service entry.
+
+-  A **mesh-external** service entry adds a service running on infrastructure
+   outside the network of the mesh. You configure a mesh-external service entry
+   so that a service inside the mesh can make API calls to an external server.
+   When you add a mesh external service entry, you don't need to configure an
```

Comment:
This sounds like an inaccurate or strange point.

Maybe a sentence like this instead:
```
You can use service entries in conjunction with an egress gateway to ensure all external services are accessed
though a single exit point.
```